lib/Runtime/Library/JavascriptArray.h

```diff
@@ -116,6 +116,7 @@ namespace Js
         static uint32 const MaxArrayLength = InvalidIndex;
         static uint32 const MaxInitialDenseLength=1<<18;
         static ushort const MergeSegmentsLengthHeuristics = 128; // If the length is less than MergeSegmentsLengthHeuristics then try to merge the segments
+        static uint64 const MaxSafeInteger = 0x1FFFFFFFFFFFFF;  // 2^53-1
```

Comment:
ECMA262 quotes the 2^53-1 literal ubiquitously without giving it a name.
Defining a constant and using it only once (in concat) doesn't seem to justify vs. using the literal itself, like the spec does.
I was searching for other occurrences and found const double Math::MAX_SAFE_INTEGER defined in MathLibrary.h
Ideally we should find a way to reuse it or at least find tie the uint64 version of the same with the double const.
Once we figure that out, I can make the change.